lib/liveman/survey/surveys.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Liveman.Survey.Surveys do
+  use Ecto.Schema
+
+  alias Liveman.Survey.Schemas.Question
+  alias Liveman.Survey.Schemas.Survey
+
+  @json_file "priv/repo/data/surveys.json"
+
+  def list_surveys do
+    survey_list_json = get_json()
```

Comment:
I don't really get how `get_json()` is returning a tuple `{:ok, json}` but it still can go through the `Enum.map` without pattern matching it here. 🤔 
Is there something I'm missing here? 😂

Reply:
Checked more, though it was working functionality, but I found that `Jason.decode!(body)` does not return a tuple, instead it returns the result directly and it cant match with ` {:ok, json}` and `with` case it returns the result if it cant match. Replaced `with` case with simple parsing code.